strings/jaro_winkler.py

```diff
@@ -0,0 +1,62 @@
+""" https://en.wikipedia.org/wiki/Jaro%E2%80%93Winkler_distance """
+import math
+
+
+def jaro_winkler(first_string: str, second_string: str) -> float:
```

Comment:
```suggestion
def jaro_winkler(str1: str, str2: str) -> float:
```
Might make the code easier to read.

Reply:
Fixed. Also found bug in implementation so fixed too.